CourseStream/CourseStream.Core/CourseStreamService.cs

```diff
@@ -23,10 +23,6 @@ public class CourseStreamAddDto
 public class CourseStreamService
 {
 	private readonly ICourseStreamRepository _repository;
-	public CourseStreamService()
```

Comment:
how this change affect local development?

Reply:
I believe this can cause issues with applying migrations

Reply:
This should be deleted in order to launch correctly application in docker. All configuration are done in startup